src/renderer/components/dialogs/Settings-Appearance.tsx

```diff
@@ -11,6 +11,7 @@ import { SettingsSelector } from './Settings'
 import { SmallSelectDialog, SelectDialogOption } from './DeltaDialog'
 import { runtime } from '../../runtime'
 import { RC_Config, Theme } from '../../../shared/shared-types'
+import { join } from 'path'
```

Comment:
don't import node dependencies in the frontend, it's not given that they work there.
its ok for join in path I think, but stuff as `fs` won't work without bad practice electron magic that weakens security.